### PR TITLE
Implement controllers for Movie, MovieSession and Authentication

### PR DESCRIPTION
**Pull request by OleksiiTromsa**

### CODE REVIEW

File: src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java [hunk 1]
```diff
@@ -20,4 +20,11 @@ public CinemaHallDaoImpl(SessionFactory sessionFactory) {
             return session.createQuery("from CinemaHall ", CinemaHall.class).getResultList();
         }
     }
+
+    @Override
+    public CinemaHall getById(Long id) {
```
Comment (jah701) [inline src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java:25]: Maybe its better to return optional on dao layer?

**Review by jah701 — APPROVED** [review]
Very well done ! :)

File: src/main/java/com/dev/cinema/controllers/AuthenticationController.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import com.dev.cinema.model.dto.UserRequestDto;
+import com.dev.cinema.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping
```
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/controllers/AuthenticationController.java]: do we need this annotation here?

**Review by RomanDubovskyi — APPROVED** [review]
remove redundant annotation in controller, rest is good

File: src/main/java/com/dev/cinema/controllers/AuthenticationController.java (hunk 2 above)
Comment (OleksiiTromsa) [reply src/main/java/com/dev/cinema/controllers/AuthenticationController.java]: Test without it and see that it is unnecessary, thanks.

**Pull request merged**